examples/gw_examples/injection_examples/eccentric_inspiral.py

```diff
@@ -12,21 +12,20 @@
 import numpy as np
 from bilby.core.utils.random import seed
 
-# Sets seed of bilby's generator "rng" to "123" to ensure reproducibility
 seed(123)
 
 duration = 64
 sampling_frequency = 256
 
 outdir = "outdir"
-label = "eccentric_GW150914"
+label = "eccentric"
 bilby.core.utils.setup_logger(outdir=outdir, label=label)
 
-
 injection_parameters = dict(
     mass_1=35.0,
     mass_2=30.0,
     eccentricity=0.1,
+    mean_per_ano=0.0,
```

Comment:
I'm a bit confused by this change, I thought `EccentricFD` doesn't take a `mean_per_ano` parameter? I see it's fixed below so perhaps we should explain this to avoid confusion.